### PR TITLE
chore; change files to lambda server

### PR DESCRIPTION
**Pull request by zoobestik**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -11,17 +12,39 @@ export const RUNTIME_CONFIG = {...getConfigFromElement(currentScript)};
...
+        break;
+      case TargetPlatform.JS:
+        url = `${this.server}/api/${version}/compiler/translate`;
+        break;
+      case TargetPlatform.JUNIT:
+        url = `${this.server}/api/${version}/compiler/test`;
+        break;
+      default:
+        console.warn(`Unknown ${target} , used by default JVM`)
+        url = `${this.server}/api/${version}/compiler/compiler`;
```
Comment (AlexanderPrendota) [inline src/config.js:33]: `/compiler/compiler` ?

File: src/config.js [hunk 2]
```diff
@@ -11,17 +12,39 @@ export const RUNTIME_CONFIG = {...getConfigFromElement(currentScript)};
...
-  get HIGHLIGHT() {
-    return `${this.server}/kotlinServer?type=highlight&runConf=`;
+
+  HIGHLIGHT(version) {
+    return `${this.server}/api/${version}/compiler/highlight`;
   },
-  get COMPLETE() {
-    return `${this.server}/kotlinServer?type=complete&runConf=`;
+  COMPLETE(version) {
+    return `${this.server}/api/${version}/compiler/complete`;
```
Comment (AlexanderPrendota) [inline src/config.js:44]: `/compiler/compiler` ? why?

Comment (zoobestik) [reply src/config.js:44]: ** complete** :)

File: src/config.js (hunk 1 above)
Comment (zoobestik) [reply src/config.js]: **fixed**

File: src/config.js (hunk 2 above)
Comment (AlexanderPrendota) [reply src/config.js:44]: oops)

File: webpack.config.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ module.exports = (params = {}) => {
   const mainEntryName = isProduction ? 'playground.min' : 'playground';
   const isServer = process.argv[1].includes('webpack-dev-server');
   const libraryName = 'KotlinPlayground';
-  const webDemoUrl = params.webDemoUrl || 'https://try.kotlinlang.org';
+  const webDemoUrl = params.webDemoUrl || 'https://api.kotlinlang.org/';
```
Comment (AlexanderPrendota) [inline webpack.config.js:11]: api -> play ?

Comment (zoobestik) [reply webpack.config.js:11]: Actually, i'm think we can use `/api/*` for internal play solution and easier for route. Does it make sense? :)

Comment (AlexanderPrendota) [reply webpack.config.js:11]: okay

**Pull request merged**